server/venueless/live/modules/world.py

```diff
@@ -0,0 +1,70 @@
+import logging
+
+from venueless.core.services.user import get_user
+from venueless.core.services.world import (
+    create_room,
+    get_room_config_for_user,
+    get_world,
+    get_world_config_for_user,
+)
+from venueless.live.exceptions import ConsumerException
+
+logger = logging.getLogger(__name__)
+
+
+class WorldModule:
+    interactive = True
```

Comment:
This seems to be unused?

Reply:
Yeah, sorry, dividing up five features into three PRs seems to have left artifacts.